io.spectra.sudoku_solver.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id" : "io.spectra.sudoku_solver",
```

Comment:
`spectra.io` points to a website in german about self publishing. Is there any relation?

I don't even see it mentionned in the upstream appdata for the homepage.

Reply:
No, there is no relation to that

Reply:
then you need to change the app-id. You are not allowed to use that.

Reply:
I'll do it right away.